README.md

```diff
@@ -222,6 +222,32 @@ app = Starlette()
 app.add_middleware(contrib.ASGIHoneybadger)
 ```
 
+### Celery
+
+A Celery extension is available for initializing and configuring Honeybadger: `honeybadger.contrib.celery.CeleryHoneybadger`. The extension adds the following information to reported exceptions:
+
+- **component**: The module that the task is defined at.
+- **action**: The name of the task.
+- **params**: The arguments and keyword arguments passed to the task.
+- **context**: A dictionary containing the following:
+  - **task_id**: The id of the current task.
+  - **retries**: The number of retries that have been attempted.
+  - **max_retries**: The maximum number of retries that will be attempted.
+
+#### Example
+
+```python
+from celery import Celery
+from honeybadger.contrib import CeleryHoneybadger
+
+app = Celery(__name__)
+app.conf.update(
+  HONEYBADGER_API_KEY= 'production',
```

Comment:
I think you mixed up the environment and API_KEY placeholders 😄

Reply:
oh right, good catch 👍

Reply:
Updated